### ISSUE TITLE
Country-wise Leaderboards?

### ISSUE THREAD

**Issue by CoderAlan24** [codewars/codewars.com#2196]
#### Leaderboard having a filter-by-country option.

**Comment by Blind4Basics** [comment]
that would require a settign in the profile of any user to know if they are ok to be displayed in that board (it's actually kind of personnal information).

**Comment by iwtga** [comment]
> that would require a setting in the profile of the users to know if they are ok to be displayed in that board (it's actually kind of personnal information).

Yeah! In that case, everyone who don't wish to be displayed can be put under 'International'(a default region)
